### ISSUE TITLE
Idea: rename "thematic data filter" to "poi data"

### ISSUE THREAD

**Issue by UJehle** [goat-community/goat#1591]
**Is your feature request related to a problem? Please describe.**
When renewing the documentation, I realized that the name and icon of the "thematic data filter" gets a bit misleading, as most of thematical data are now being found in the "other layers" tab.

**Describe the solution you'd like**
 I thought we could rename it to something like "poi data" and maybe also replacing the icon? 
![pois](https://user-images.githubusercontent.com/49870195/194235791-193b9a4d-3fec-49bd-81b5-d6c1d176d2eb.png)

**Comment by EPajares** [comment]
POI is a bit limiting. As we also have AOIs or we might want to add more stuff there. But I agree thematic data is the wrong term. It is sort of legacy from times when we had not other layers. In fact they are "Destinations" or "Opportunities" but these terms might not be very understandable as well.

**Comment by EPajares** [comment]
@AnaSasso in case you have any ideas your thoughts are warmly welcome

**Comment by UJehle** [comment]
Ya that's true.. in theory, the parks e.g. are AOIs but most people still call them "POIs".
"Destinations" is also a good idea!

**Comment by AnaSasso** [comment]
I agree that the icon of a filter might be a bit misleading. I wonder how much of the interface are you willing to update with the migration aforementioned. But for now, I'd like to suggest standardizing the nomenclatures. 
As the toaster says "Points of interest" maybe should always be referred to as such.
About the icon, because when you select a category it show the POI as markers on the map, the icon on the right bar could also be a marker.
<img width="687" alt="Screenshot 2022-10-21 at 08 36 52" src="https://user-images.githubusercontent.com/116002158/197187732-fc3b6260-238c-45f0-866a-f8af1809d386.png">
